pkg/cmd/pipelinerun/log_test.go

```diff
@@ -0,0 +1,251 @@
+// Copyright © 2019 The tektoncd Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package pipelinerun
+
+import (
+	"bytes"
+	"github.com/tektoncd/cli/pkg/cmd/taskrun"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/jonboulle/clockwork"
+	"github.com/knative/pkg/apis"
+	"github.com/tektoncd/cli/pkg/cli"
+	"github.com/tektoncd/cli/pkg/logs"
+	tu "github.com/tektoncd/cli/pkg/test"
+	cb "github.com/tektoncd/cli/pkg/test/builder"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	"github.com/tektoncd/pipeline/pkg/reconciler/v1alpha1/pipelinerun/resources"
+	"github.com/tektoncd/pipeline/test"
+	tb "github.com/tektoncd/pipeline/test/builder"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func Test_command_has_pipelinerun_arg(t *testing.T) {
+	c := Command(&tu.Params{})
+
+	_, err := tu.ExecuteCommand(c, "logs", "-n", "ns")
+
+	if err == nil {
+		t.Error("Expecting an error but it's empty")
+	}
+}
+
+func Test_pipelinerun_not_found(t *testing.T) {
+	pr := []*v1alpha1.PipelineRun{
+		tb.PipelineRun("output-pipeline-1", "ns",
+			tb.PipelineRunLabel("tekton.dev/pipeline", "output-pipeline-1"),
+			tb.PipelineRunStatus(
+				tb.PipelineRunStatusCondition(apis.Condition{
+					Status: corev1.ConditionFalse,
+					Reason: resources.ReasonFailed,
+				}),
+			),
+		),
+	}
+	cs, _ := test.SeedTestData(test.Data{PipelineRuns: pr})
+	p := &tu.Params{Tekton: cs.Pipeline, Kube: cs.Kube}
+
+	c := Command(p)
+	got, _ := tu.ExecuteCommand(c, "logs", "output-pipeline-2", "-n", "ns")
+	expected := msgPRNotFoundErr + " : pipelineruns.tekton.dev \"output-pipeline-2\" not found \n"
+
+	if d := cmp.Diff(expected, got); d != "" {
+		t.Errorf("Unexpected output mismatch: \n%s\n", d)
+	}
+}
+
+func Test_print_pipelinerun_logs(t *testing.T) {
+	var (
+		pipelineName = "output-pipeline"
```

Comment:
Same comment as the other PR, we can inline those :wink:

Reply:
I tried removing it later working on tests I realized indeed it helped me to figure what values are being passed actually. :sweat_smile: 

I do understand it may appear as extra noise but it helps I think. Shall we please keep this :smiley: ?

Reply:
yes we can :wink: